src/parse/parser.rs

```diff
@@ -1280,6 +1280,17 @@ where
             )?);
         } else {
             debug!("None");
+            if needs_eq && min_vals_zero {
+                // OPTION with missing optional value?
+                if !opt.default_missing_vals.is_empty() {
+                    // debug!("Parser::add_value:iter:{}: has default vals", arg.name);
```

Comment:
Remove comment

Reply:
`// debug!...` removed.

Reply:
Actually, with refactor die to `cargo clippy` complaint ... both comments are now gone.